specificatie/unresolved/openapi.yaml

```diff
@@ -758,6 +758,14 @@ components:
           type: "string"
         aard:
           $ref: "#/components/schemas/Waardelijst"
+        beslaglegger:
```

Comment:
Technisch gezien is dit goed, maar als je alleen een identificatie wil doorgeven, is het voldoende om dit te definieren als beslagleggerIdentificatie van type string